Resources/Maps/_Sunrise/Shuttles/DSO/ERT/ert_medium_gamma.yml

```
@@ -0,0 +1,2995 @@
+- proto: CableApcStack
+  entities:
+  - uid: 30
+    components:
+    - type: Transform
+      parent: 27
+    - type: Physics
+      canCollide: False
+    - type: InsideEntityStorage
+- proto: CableHV
+  entities:
+  - uid: 62
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)